src/pkg/control/options.go

```diff
@@ -99,7 +99,7 @@ type Toggles struct {
 	DisableSlidingWindowLimiter bool `json:"disableSlidingWindowLimiter"`
 
 	// see: https://github.com/alcionai/corso/issues/4688
-	UseDeltaTree bool `json:"useDeltaTree"`
+	UseOldDeltaProcess bool `json:"useDeltaTree"`
```

Comment:
@ryanfkeepers we should update the json tag for this

Reply:
doh!  Thanks, i'll get that.